src/cltk/dependency/spacy_dep.py

```diff
@@ -31,34 +27,42 @@ class SpacyWrapper:
 
     def __init__(self,
                  language: str,
+                 spacy_language: spacy.Language = None,
```

Comment:
Also typing is misleading, if it's nullable then you should probably type it as `Optional[spacy.Language]`